packages/editor/src/components/widgets/code-editor/ScriptInput.tsx

```diff
@@ -1,17 +1,26 @@
+import './ScriptInput.css';
 import { Input } from '../input';
 import SingleLineCodeEditor, { CodeEditorInputProps } from './SingleLineCodeEditor';
-import { useCodeEditorOnFocus } from './useCodeEditorOnFocus';
+import { useCodeEditorOnFocus, useModifyEditor } from './useCodeEditor';
+import { Browser, useBrowser } from '../../../components/browser';
 
 const ScriptInput = (props: CodeEditorInputProps) => {
-  const { active, onBlur, nonFocusProps } = useCodeEditorOnFocus(props);
-  if (active) {
-    return (
-      <div onBlur={onBlur}>
-        <SingleLineCodeEditor {...props} />
-      </div>
-    );
-  }
-  return <Input {...nonFocusProps} />;
+  const { isFocusWithin, focusWithinProps } = useCodeEditorOnFocus();
+  const browser = useBrowser();
+  const { setEditor, modifyEditor } = useModifyEditor();
+
+  return (
+    <div className='script-input' {...focusWithinProps}>
+      {isFocusWithin || browser.open ? (
+        <>
+          <SingleLineCodeEditor {...props} onMountFuncs={[setEditor]} />
+          <Browser {...browser} types={['attribute', 'cms']} accept={modifyEditor} />
```

Comment:
maybe you could/should generalize that part 'types=attr+cms' ... as it is now equally stated on all script editors.

Reply:
Yes not part of this pr. I will address that as soon as I implement a better UI with real data 👍